boards/st/stm32f746g_disco/stm32f746g_disco.dts

```diff
@@ -24,6 +24,7 @@
 		zephyr,dtcm = &dtcm;
 		zephyr,flash-controller = &n25q128a1;
 		zephyr,display = &ltdc;
+		zephyr,touch = &ft5336;
```

Comment:
I like this idea very much, this could be way forward to remove the lvgl-pointer-input from the intree board definitions. Initially this was done for ease of use coming from the old kscan-input for LVGL. 

I imagine adding a flag `CONFIG_LV_Z_POINTER_FROM_CHOSEN` which defaults to true and depends on LVGL. As a whole this will still get users a "just works" experience with LVGL while removing module dependencies from intree boards. Thanks for the impulse @kartben

Reply:
I believe this change was applied.